tests/Util/ApiLoaderTest.php

```diff
@@ -168,4 +162,14 @@ protected function getLoader($cacheMock = false)
 
         return $loader;
     }
+
+    protected function getMockCacheInterface()
+    {
+        $methods = ['get', 'set', 'has', 'delete', 'getMultiple', 'setMultiple', 'clear', 'deleteMultiple'];
+        $mock = $this->getMockBuilder(CacheInterface::class)
+            ->setMethods($methods)
```

Comment:
setMethods() is going to be deprecated/removed - https://github.com/sebastianbergmann/phpunit/issues/4775

I think you're supposed to use addMethods() instead? https://phpunit.readthedocs.io/en/9.5/test-doubles.html